Makefile

```diff
@@ -93,18 +90,12 @@ push_translations:
 	./node_modules/@edx/reactifex/bash_scripts/put_comments_v3.sh
 
 
-ifeq ($(OPENEDX_ATLAS_PULL),)
-# Pulls translations from Transifex.
-pull_translations:
-	tx pull -t -f --mode reviewed --languages=$(transifex_langs)
-else
 # Experimental: OEP-58 Pulls translations using atlas
 pull_translations:
 	rm -rf src/i18n/messages
 	cd src/i18n/ \
-	  && atlas pull --filter=$(transifex_langs) translations/studio-frontend/src/i18n/messages:messages
+	  && atlas pull translations/studio-frontend/src/i18n/messages:messages
 	$(generate_supported_langs) $(transifex_langs)
```

Comment:
The problem with this line is that it depends on the `transifex_langs` hard-coded variable, which is no longer used.

One solution is to update the `src/i18n/scripts/generateSupportedLangs.js` script to generate the list based on the following algorithm:

 - list all files in src/i18n/messages:
 
   ```
   studio-frontend $ ls src/i18n/messages | cat
    ar.json
    de_DE.json
    de.json
    es_419.json
    ...
   ```

 - Use the list of languages from the files, instead of the `process.argv`: https://github.com/openedx/studio-frontend/blob/2116cdb5b3009f84ecddcbd43a0839b3a27c3d4f/src/i18n/scripts/generateSupportedLangs.js#L100

Reply:
@OmarIthawi fixes got applied, you can check please.

Reply:
Thanks @hopewise, I've checked the fixes and it looks good. It appears that we don't have tests for the new changes which I think is important to have.

Would you mind adding tests and fix the [failed nodejs_CI tests build](https://github.com/openedx/studio-frontend/actions/runs/6681397032/job/18155508950?pr=383) as well?

Reply:
- This pull request needs to be rebased over https://github.com/openedx/studio-frontend/pull/389 once the latter is merged.